app/main/views/suppliers.py

```diff
@@ -185,6 +195,12 @@ def create_supplier():
         ['contactInformation']
     )
 
+    # duns number does not have a unique constraint due to nulls
```

Comment:
I'm not sure if we need to do this, I think NULLs are exempt for the unique constraints. From the postgres docs: "even in the presence of a unique constraint it is possible to store duplicate rows that contain a null value in at least one of the constrained columns." http://www.postgresql.org/docs/9.5/interactive/ddl-constraints.html